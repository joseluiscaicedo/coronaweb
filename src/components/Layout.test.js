import React from 'react';
import '@testing-library/jest-dom/extend-expect'
import { render } from '@testing-library/react'
import Layout from './Layout';


test('Generates Snapshot for Layout Component', () => {
  const { container } = render(
    <Layout />
  );
  expect(container.firstChild).toMatchSnapshot();
});