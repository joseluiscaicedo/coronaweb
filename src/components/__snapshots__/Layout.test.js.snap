// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Generates Snapshot for Layout Component 1`] = `
<header
  aria-label="header title"
  class="header"
>
  <a
    aria-label="goToWorldMap"
    href="/worldmap"
  >
    <img
      alt="coronaweb logo"
      class="header__img"
      src="https://res.cloudinary.com/josecaicedo/image/upload/v1627426970/coronaweb/logo_yn8r2a.png"
    />
  </a>
  <div
    class="header__menu"
  >
    <div
      class="header__menu"
    >
      <img
        alt="user Icon"
        id="menu__img"
        src="https://res.cloudinary.com/josecaicedo/image/upload/v1627426970/coronaweb/user-icon_o089f9.png"
      />
      <button
        class="ButtonLogin"
        id="ButtonLogin"
        type="button"
      >
        Login
      </button>
    </div>
  </div>
</header>
`;
